src/libical-glib/api/i-cal-geo.xml

```diff
@@ -51,28 +51,28 @@
     <method name="i_cal_geo_get_lat" corresponds="CUSTOM" kind="get" since="1.0">
 	<parameter type="ICalGeo *" name="geo" comment="The #ICalGeo to be queried."/>
         <returns type="gdouble" comment="The lat." />
-        <comment xml:space="preserve">Get the lat of #ICalGeo.</comment>
+        <comment xml:space="preserve">Gets the lat of #ICalGeo.</comment>
         <custom>	g_return_val_if_fail (geo != NULL, 0);
 	return ((struct icalgeotype *)i_cal_object_get_native ((ICalObject *)geo))->lat;</custom>
     </method>
     <method name="i_cal_geo_set_lat" corresponds="CUSTOM" kind="set" since="1.0">
         <parameter type="ICalGeo *" name="geo" comment="The #ICalGeo to be set."/>
         <parameter type="gdouble" name="lat" comment="The lat."/>
-        <comment>Set the lat of #ICalGeo.</comment>
+        <comment>Sets the lat of #ICalGeo.</comment>
         <custom>	g_return_if_fail (geo != NULL &amp;&amp; I_CAL_IS_GEO (geo));
 	((struct icalgeotype *)i_cal_object_get_native ((ICalObject *)geo))->lat = lat;</custom>
     </method>
     <method name="i_cal_geo_get_lon" corresponds="CUSTOM" kind="get" since="1.0">
 	<parameter type="ICalGeo *" name="geo" comment="The #ICalGeo to be queried."/>
         <returns type="gdouble" comment="The lon." />
-        <comment xml:space="preserve">Get the lon of #ICalGeo.</comment>
+        <comment xml:space="preserve">Gets the lon of #ICalGeo.</comment>
```

Comment:
"lon"?  let's say longitude

Reply:
Done. And all the other references to lat and lon in this file too.